### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by Savina05**
## Overview

- Add dashboard route
- Add dashboard to navbar
- Create dashboard view
- Display all tasks in dashboard (both as creator and helper)
- Add modal for each task
- Add edit and delete buttons for creator tasks (logic works)
- Add undo application button for helper tasks (logic works)

## Notes

I wasn't able to add the carousel, we can do it later if you want.
I wasn't able to refactor the helper modal (if I create a partial for it, the undo button stops working).

### CODE REVIEW

**Comment by gabrielecanepa** [comment]
[Tasks dashboard](https://trello.com/c/eduvz5xm/20-tasks-dashboard)

File: app/views/pages/dashboard.html.erb [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<div class="container">
+  <br>
+  <h1 class="text-center">Hi <%= current_user.first_name.capitalize %>!</h1>
+  <br>
+  <h4>Tasks you created</h4>
+  <div class="row">
+    <div class="d-flex justify-content-around flex-wrap">
+      <% @created_tasks.each do |task| %>
```
Comment (gabrielecanepa) [inline app/views/pages/dashboard.html.erb:8]: Add a condition here:
```ruby
<% if @created_tasks.empty? %>
  <p>You haven't created any task yet</p>
<% else %>
  <% @created_tasks.each do |task| %>
    <# ... %>
  <% end %>
<% end %>
```

File: app/views/pages/dashboard.html.erb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  </div>
+
+  <br>
+  <hr>
+  <br>
+
+  <h4>Tasks you applied for</h4>
+  <div class="row">
+    <div class="d-flex justify-content-around flex-wrap">
+      <% @helper_tasks.each do |task| %>
```
Comment (gabrielecanepa) [inline app/views/pages/dashboard.html.erb:47]: Same as the other ☝️

File: app/views/pages/dashboard.html.erb (hunk 1 above)
Comment (Savina05) [reply app/views/pages/dashboard.html.erb]: I added the condition

File: app/views/pages/dashboard.html.erb (hunk 2 above)
Comment (Savina05) [reply app/views/pages/dashboard.html.erb]: Added also the condition here

**Pull request merged**